proto/rollup/v1/msg.proto

```diff
@@ -22,15 +22,16 @@ message ApplyL1TxsRequest {
 
 message ApplyL1TxsResponse {}
 
-// MsgInitiateWithdrawal defines a message for all L2 withdrawal txs
+// TODO: add validations in the x/rollup module to ensure that the target address and gas limit are valid before burning the users ETH
+// InitiateWithdrawalRequest defines a message for all L2 withdrawal txs
 message InitiateWithdrawalRequest {
   option (cosmos.msg.v1.signer) = "sender";
   // The cosmos address of the user who wants to withdraw from L2.
   string sender = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
   // The ethereum address on L1 that the user wants to withdraw to.
   string target = 2;
-  // The amount of ETH that the user wants to withdraw.
-  string amount = 3 [
```

Comment:
I renamed `amount` to `value` to have parity with the Ethereum withdrawal tx type.

Reply:
Clarifying: "Ethereum withdrawal tx type" here refers to the OP stack [withdrawal event type](https://specs.optimism.io/protocol/withdrawals.html#the-l2tol1messagepasser-contract) and [L1 finalizing tx type](https://specs.optimism.io/protocol/withdrawals.html#withdrawal-verification-and-finalization)?

Reply:
There is a universal `WithdrawalTransaction` [struct](https://github.com/ethereum-optimism/optimism/blob/08daf8dbd38c9ffdbd18fc9a211c227606cdb0ad/packages/contracts-bedrock/src/libraries/Types.sol#L55-L69) created in OP Stack's `Types.sol` that is used to build the initiating, proving, and finalizing transactions on both L2 and L1. This is the struct that I'm referring to here